tests/infra_test.go

```diff
@@ -366,6 +370,53 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
 		)
 	})
 
+	Describe("tls configuration", func() {
+
+		DescribeTable("should be propagated", func(labelSelector string, featureGate string) {
```

Comment:
This could be simple `It`

Reply:
The `DescribeTable` has many entry and cannot be an `It`. The first `Describe` IMHO can remain as is since we are describing a different feature of the infrastructure. WDYT?

Reply:
You could just query all the pods at the same time. This is not mandatory.

Reply:
Right, got it. Thanks. In order to reduce the overhead I will change the code as you suggested. Thank you!